src/IssueLabeler/AzureSdkIssueLabelerService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Hubbup.MikLabelModel;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace IssueLabeler
+{
+    public class AzureSdkIssueLabelerService : IssueLabelerFunctionBase
+    {
+        public AzureSdkIssueLabelerService(ILabelerLite labeler, IModelHolderFactoryLite modelHolderFactory, IConfiguration config)
+            : base(labeler, modelHolderFactory, config)
+        {
+        }
+
+        [FunctionName("AzureSdkIssueLabelerService")]
+        public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Function, "POST", Route = null)] HttpRequest request, ILogger log)
```

Comment:
should we return the model type? I think it will create an object response by default and return json if the accept header is text/json

Reply:
Interesting.   I didn't realize that was a thing.   Unfortunately, it seems to ignore the Accept header altogether and respond with `application/json` regardless... but it does offer better clarity on what's coming back.   Updated.